google/resource_sql_database_instance_test.go

```diff
@@ -568,7 +606,7 @@ func testAccCheckGoogleSqlDatabaseInstanceEquals(n string,
 			return fmt.Errorf("Error settings.crash_safe_replication mismatch, (%s, %s)", server, local)
 		}
 
-		server = strconv.FormatBool(instance.Settings.StorageAutoResize)
+		server = strconv.FormatBool(*instance.Settings.StorageAutoResize)
```

Comment:
I'm seeing panics at this line when running tests, can you check it out?

Reply:
Having a look at this now- do you have the test that caused the fatal? Would speed up things on my end, as the few I've been running do not!

Reply:
Think I've got a fix for this, will update the PR once I've confirmed tests pass.